tests/integration/test_refreshable_mat_view/test.py

```diff
@@ -0,0 +1,864 @@
+    node.query("DROP TABLE IF EXISTS test_rmv ON CLUSTER test_cluster")
+
+
+@pytest.mark.parametrize(
+    "append",
+    [True, False],
+)
+# @pytest.mark.parametrize("on_cluster", [True, False])
+@pytest.mark.parametrize(
+    "empty",
+    [True, False],
+)
+@pytest.mark.parametrize(
+    "to_clause",
+    [(None, "tgt1", "tgt1"), ("Engine MergeTree ORDER BY tuple()", None, "test_rmv")],
+)
+def test_real_wait_refresh(
+    fn2_setup_tables,
+    started_cluster,
+    append,
+    # on_cluster,
+    empty,
+    to_clause,
+):
+    table_clause, to_clause_, tgt = to_clause
+
+    create_sql = CREATE_RMV_TEMPLATE.render(
+        table_name="test_rmv",
+        # if_not_exists=if_not_exists,
+        refresh_interval="EVERY 8 SECOND",
+        to_clause=to_clause_,
+        table_clause=table_clause,
+        select_query="SELECT a as a, b FROM src1",
+        with_append=append,
+        # on_cluster="test_cluster" if on_cluster else None,
+        empty=empty
+        # settings={'setting1':'value1', 'setting2': 'value2'},
+    )
+    node.query(create_sql)
+    rmv = get_rmv_info(node, "test_rmv")
+
+    append_expected_rows = 0
+    if not empty:
+        append_expected_rows += 2
+
+    if empty:
+        expect_rows(0, table=tgt)
+    else:
+        expect_rows(2, table=tgt)
+
+    rmv2 = get_rmv_info(
+        node,
+        "test_rmv",
+        condition=lambda x: x["last_refresh_time"] == rmv["next_refresh_time"],
+        # wait for refresh a little bit more than 8 seconds
+        max_attempts=10,
+        delay=1,
+    )
+
+    if append:
+        append_expected_rows += 2
+        expect_rows(append_expected_rows, table=tgt)
+    else:
+        expect_rows(2, table=tgt)
+
+    assert rmv2["exception"] is None
+    assert rmv2["refresh_count"] == 2
+    assert rmv2["status"] == "Scheduled"
+    assert rmv2["last_refresh_result"] == "Finished"
+    # assert rmv2["progress"] == , 1rmv2
+    assert rmv2["last_success_time"] == rmv["next_refresh_time"]
+    assert rmv2["last_refresh_time"] == rmv["next_refresh_time"]
+    assert rmv2["retry"] == 0
+    assert rmv2["read_rows"] == 2
+    assert rmv2["read_bytes"] == 24
+    assert rmv2["total_rows"] == 0
+    assert rmv2["total_bytes"] == 0
+    assert rmv2["written_rows"] == 0
+    assert rmv2["written_bytes"] == 0
+    assert rmv2["result_rows"] == 0
+    assert rmv2["result_bytes"] == 0
+
+    node.query("SYSTEM STOP VIEW test_rmv")
+    time.sleep(10)
+    rmv3 = get_rmv_info(node, "test_rmv")
+    # no refresh happen
+    assert rmv3["status"] == "Disabled"
+
+    del rmv3["status"]
+    del rmv2["status"]
+    assert rmv3 == rmv2
+
+    node.query("SYSTEM START VIEW test_rmv")
+    time.sleep(1)
+    rmv4 = get_rmv_info(node, "test_rmv")
+
+    if append:
+        append_expected_rows += 2
+        expect_rows(append_expected_rows, table=tgt)
+    else:
+        expect_rows(2, table=tgt)
+
+    assert rmv4["exception"] is None
+    assert rmv4["refresh_count"] == 3
```

Comment:
https://github.com/ClickHouse/ClickHouse/pull/60669 removes `refresh_count` (because it seems confusing that it's only since server startup, in uncoordinated mode).

Reply:
Removed it too